src/shared/util/phone-num-validation.ts

```diff
@@ -54,11 +54,21 @@ export const isMobilePhoneNumber = (mobileNumber: string): boolean => {
  */
 export const isHomePhoneNumber = (phoneNum: string): boolean => {
   const parsedNumber = parsePhoneNumberFromString(phoneNum)
+
   if (!parsedNumber) return false
 
+  // For SG numbers only check if it starts with 3 or 6 and has 8 digits
+  if (parsedNumber.countryCallingCode === '65') {
+    return (
+      isPhoneNumber(phoneNum) &&
+      // Regex checks if the national number starts with 3 or 6, and is of
+      // length 8.
```

Comment:
where do these rules come from? eg how do we know that 3 is allowed? and should we document the source?

Reply:
added